x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/enrich.tsx

```diff
@@ -0,0 +1,264 @@
+    deserializer: (v) => (typeof v === 'number' && !isNaN(v) ? v : 1),
+    serializer: (v) => {
+      const n = parseInt(v, 10);
+      return n === 1 ? undefined : n;
+    },
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.enrichForm.maxMatchesFieldLabel', {
+      defaultMessage: 'Maximum matches (optional)',
+    }),
+    helpText: i18n.translate(
+      'xpack.ingestPipelines.pipelineEditor.enrichForm.maxMatchesFieldHelpText',
+      {
+        defaultMessage:
+          'Number of matching enrich documents to include in the target field. Accepts 1–128.',
+      }
+    ),
+    validations: [
+      {
+        validator: (v) => {
+          if (v.value /* value is a string here */) {
+            return maxMatchesValidators.max(v) ?? maxMatchesValidators.min(v);
+          }
+        },
+      },
+    ],
+  },
+
+  shape_relation: {
+    type: FIELD_TYPES.SELECT,
+    defaultValue: 'INTERSECTS',
+    deserializer: String,
+    serializer: (v) => (v === 'INTERSECTS' ? undefined : v),
+    label: i18n.translate(
+      'xpack.ingestPipelines.pipelineEditor.enrichForm.shapeRelationFieldLabel',
+      {
+        defaultMessage: 'Shape relation (optional)',
+      }
+    ),
+  },
+};
+
+export const Enrich: FunctionComponent = () => {
+  const { services } = useKibana();
+  const esDocUrl = services.documentation.getEsDocsBasePath();
+  return (
+    <>
+      <FieldNameField
+        helpText={i18n.translate(
+          'xpack.ingestPipelines.pipelineEditor.enrichForm.fieldNameHelpText',
+          {
+            defaultMessage:
+              'Field used to match incoming documents to enrich documents. Field values are compared to the match field set in the enrich policy.',
+          }
+        )}
+      />
+
+      <UseField
+        config={{
+          ...fieldsConfig.policy_name,
+          helpText: (
+            <FormattedMessage
+              id="xpack.ingestPipelines.pipelineEditor.enrichForm.policyNameHelpText"
+              defaultMessage="Name of the {enrichPolicyLink}."
+              values={{
+                enrichPolicyLink: (
+                  <EuiLink external target="_blank" href={esDocUrl + '/ingest-enriching-data.html'}>
+                    {i18n.translate(
+                      'xpack.ingestPipelines.pipelineEditor.enrichForm.policyNameHelpText.enrichPolicyLink',
+                      { defaultMessage: 'enrich policy' }
+                    )}
+                  </EuiLink>
+                ),
+              }}
+            />
+          ),
+        }}
+        component={Field}
+        path="fields.policy_name"
+      />
+
+      <TargetField
+        label={i18n.translate('xpack.ingestPipelines.pipelineEditor.enrichForm.targetFieldLabel', {
+          defaultMessage: 'Target field',
+        })}
+        helpText={i18n.translate(
+          'xpack.ingestPipelines.pipelineEditor.enrichForm.targetFieldHelpText',
+          {
+            defaultMessage: 'Field used to contain enrich data.',
```

Comment:
Period isn't needed.
```suggestion
            defaultMessage: 'Field used to contain enrich data',
```